modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/feather.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/tickle_feather
 	name = "tickling feather"
-	desc = "Feather that can be used for tickling or even torturing. Have fun!"
+	desc = "A feather that can be used for tickling or even torture. Have fun!"
```

Comment:
`A rather ticklish feather that can be used both in mirth and malice. Have fun!`